packages/blocks/src/api/raw-handling/image-corrector.native.js

```diff
@@ -0,0 +1,11 @@
+
+export default function( node ) {
```

Comment:
Can we add some code comments about what this function is doing? I couldn't understand its purpose.

Reply:
Thank you